wire/msgcftypes.go

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2017 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package wire
+
+import "io"
+
+// MsgCFTypes is the cftypes message.
+type MsgCFTypes struct {
+	NumFilters       uint8
```

Comment:
I don't think we actually need to include this value _within_ the struct itself. As it's only needed on the wire in order to indicate how many bytes more the parser should read.

Reply:
I've removed this and made it a varint so we can support up to 256 instead of 255 filter types. In almost every foreseeable situation, it should still only use 1 byte for the count on the wire (assuming at least some filter types are obsoleted if and when we have over 200 types created).